### ISSUE TITLE
simplest OAuth2 Authorization Server blows up with Client 

### ISSUE THREAD

**Issue by lennane** [IdentityServer/Thinktecture.IdentityServer3#924]
Self Hosting server is running and services https://localhost:44333 
But "Client" example fails on line
return client.RequestClientCredentialsAsync("api1").Result;
with AggregatedException.
I have localhost SSL in Personal and TrustedRoot
Code simply copied from GitHub for Client and Scopes.
Any help would be appreciated.

**Comment by leastprivilege** [comment]
You need to check the details of the AggregrateException. Normal .NET - learn to help yourself.

**Comment by KeithBarrows** [comment]
I have looked when I get this error.  I setup 2 new scopes and 2 new users.  When I run the samples they pass but when I run the new users they fail with a {"Response status code does not indicate success: 403 (Forbidden)."}  Nothing else indicating what went wrong, the break point in the REST service is never hit, etc.  I am running with RequireSsl = false, all Users have access to all Scopes.

Stack Trace:
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at Client.Program.CallApi(TokenResponse response) in ...\Simplest OAuth2 Walkthrough\Client\Program.cs:line 33
   at Client.Program.Main(String[] args) in ...\Simplest OAuth2 Walkthrough\Client\Program.cs:line 22
   at System.AppDomain._nExecuteAssembly(RuntimeAssembly assembly, String[] args)
   at Microsoft.VisualStudio.HostingProcess.HostProc.RunUsersAssembly()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()

Turns out that the RequiredScopes was not set with the new Scopes in the REST Service Configuration method!  Added them there and my 403 went away.
